spring-cloud-schema-registry-server/pom.xml

```diff
@@ -43,6 +47,18 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>com.spotify</groupId>
+                <artifactId>dockerfile-maven-plugin</artifactId>
+                <version>1.4.13</version>
```

Comment:
Replace the hardcoded version with a property (e.g. `<version>${dockerfile-maven-plugin.version}</version>`) and define the `dockerfile-maven-plugin.version` property in the [parent's POM property section](https://github.com/spring-cloud/spring-cloud-schema-registry/blob/master/pom.xml#L24-L30)